uc-crux-llvm/src/UCCrux/LLVM/Equivalence/Config.hs

```diff
@@ -0,0 +1,34 @@
+{-
+Module       : UCCrux.LLVM.Equivalence.Config
+Description  : Configuration for crash-equivalence checking
+Copyright    : (c) Galois, Inc 2021
+License      : BSD3
+Maintainer   : Langston Barrett <langston@galois.com>
+Stability    : provisional
+-}
+
+module UCCrux.LLVM.Equivalence.Config
+  ( OrderOrEquivalence(..),
+    EquivalenceConfig(..)
+  )
+where
+
+import UCCrux.LLVM.Newtypes.FunctionName (FunctionName)
+
+-- | Check for crash ordering or crash equivalence? Ordering asserts that one
+-- function\'s undefined behaviors are a subset of the other\'s, equivalence
+-- asserts that two functions have the exact same set of undefined behaviors.
+data OrderOrEquivalence
+  = Order
+  | Equivalence
```

Comment:
The constructors here are not related to the Haskell `Eq` and `Ord` typeclasses, but instead refer to how `--crash-equivalence` works. In particular, `--crash-order` checks that one version of a function has fewer crashes than the other, whereas `--crash-equivalence` checks that both versions have the same crashes. The former is a partial order and the latter is its corresponding equivalence relation.

Reply:
I agree that the domain of the concerns is different, but from a conceptual perspective, fewer (is it strictly fewer, or fewer or the same number?) crashes is a stronger check than equivalence, so from an `Ord` perspective I would expect it to be "greater" than `Equivalence`.  I suspect the `Ord` here is mostly due to some requirement elsewhere like `Map` or `Set`, but I'm suggesting that reversing the constructor order might match the conceptual "ordering" of the strength of verification in case some client applies that concept.

The above is a better explanation of my original thinking.  It may be misguided, and/or feel free to disagree.

Reply:
Ah, I see it in just the opposite way: 5 is less than or equal to 10 (ordering), but is not equal to 10 (equivalence). So I see equivalence as the stronger condition! Since reasonable arguments can be made in either direction, I'll add a comment to the effect of "don't depend on the `Ord` instance being some particular way".